### PR TITLE
New Device (Matter Switch) Cync Paddle Dimmer

### PR DESCRIPTION
**Pull request by lelandblue**
Check all that apply

# Type of Change

- [X] WWST Certification Request
     - If this is your first time contributing code:
          - [ ] I have reviewed the README.md file
          - [ ] I have reviewed the CODE_OF_CONDUCT.md file
          - [ ] I have signed the CLA
     - [ ] I plan on entering a WWST Certification Request or have entered a request through the WWST Certification console at developer.smartthings.com
- [ ] Bug fix
- [ ] New feature
- [ ] Refactor

# Checklist

- [X ] I have performed a self-review of my code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have verified my changes by testing with a device or have communicated a plan for testing
- [ ] I am adding new behavior, such as adding a sub-driver, and have added and run new unit tests to cover the new behavior

# Description of Change
New fingerprint addition for a new Matter Device, the Cync Paddle Dimmer. 
I was unable to verify this devices Matter Device Type in the DCL. 

# Summary of Completed Tests

### CODE REVIEW

File: drivers/SmartThings/matter-switch/fingerprints.yml [hunk 1]
```diff
@@ -180,6 +180,11 @@ matterManufacturer:
     vendorId: 0x1339
     productId: 0x006B
     deviceProfileName: light-color-level-2000K-7000K
+  - - id: "4921/125"
```
Comment (nickolas-deboom) [inline drivers/SmartThings/matter-switch/fingerprints.yml:183]: It looks like there is an extra '-' on this line

**Comment by nickolas-deboom** [comment]
Hi, I think this PR will be good to go once the formatting issue I mentioned is fixed!

File: drivers/SmartThings/matter-switch/fingerprints.yml (hunk 1 above)
Comment (lelandblue) [reply drivers/SmartThings/matter-switch/fingerprints.yml]: Thank you!!! I have resolved this @nickolas-deboom !

**Review by nickolas-deboom — APPROVED** [review]

File: drivers/SmartThings/matter-switch/fingerprints.yml (hunk 1 above)
Comment (nickolas-deboom) [reply drivers/SmartThings/matter-switch/fingerprints.yml]: Thank you! I approved the PR

**Pull request merged**